syntax = "proto2";

message message_detection{
    required string result = 1;
}

